### ISSUE TITLE
Inlined meta(s) do not work with field macro anymore

### ISSUE THREAD

**Issue by tlvenn** [absinthe-graphql/absinthe#958]
Trying to upgrade our app / schema to the latest 1.5 release (still using 1.4 for now) and I got a compilation error in our schema regarding the usage of meta with field such as:

```
input do
  field(:country_id, :id, meta: [type: Jumpn.SupportedCountry])
end
```
 
The error:
```
KeyError) key :meta not found in: %Absinthe.Blueprint.Schema.FieldDefinition{__private__: [], __reference__: %{location: %{file: ".../activity/schema.ex", line: 29}, module: Jumpn.Schema.Activity}, arguments: [], complexity: nil, config: nil, default_value: nil, deprecation: nil, description: nil, directives: [], errors: [], flags: %{}, function_ref: nil, identifier: :country_id, middleware: [], module: Jumpn.Schema.Activity, name: "country_id", source_location: nil, triggers: [], type: :id}
    (stdlib 3.13) :maps.update(:meta, [type: Jumpn.SupportedCountry], %Absinthe.Blueprint.Schema.FieldDefinition{__private__: [], __reference__: %{location: %{file: ".../activity/schema.ex", line: 29}, module: Jumpn.Schema.Activity}, arguments: [], complexity: nil, config: nil, default_value: nil, deprecation: nil, description: nil, directives: [], errors: [], flags: %{}, function_ref: nil, identifier: :country_id, middleware: [], module: Jumpn.Schema.Activity, name: "country_id", source_location: nil, triggers: [], type: :id})
    lib/absinthe/blueprint/schema/field_definition.ex:7: anonymous fn/2 in Absinthe.Blueprint.Schema.FieldDefinition.__struct__/1
```

Might be related with #928

**Comment by jeroenvisser101** [comment]
It's for sure related to #927

**Comment by tlvenn** [comment]
Hi @benwilson512 , is there any chance you would have a little bit of spare time to look into this ? Thanks a lot in advance.

**Comment by tlvenn** [comment]
Fixed with #973
